File: 2015/src/test/java/Day6Test.java
import nl.sbmf21.aoc15.Aoc;
import nl.sbmf21.aoc15.days.Day6;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class Day6Test {

    private final Day6 day = new Day6(new Aoc(), 6);

    @Test
    public void testPart1Output() {
        assertEquals(543903, (int) day.part1());
    }

    @Test
    public void testPart2Output() {
        assertEquals(14687245, (int) day.part2());
    }
}

File: 2020/kotlin/src/test/kotlin/Day6Test.kt
import nl.sbmf21.aoc20.Aoc
import nl.sbmf21.aoc20.days.Day6
import org.junit.Test
import kotlin.test.assertEquals

class Day6Test {

    private var day = Day6(Aoc())

    @Test
    fun testPart1Output() = assertEquals(6903, day.part1())

    @Test
    fun testPart2Output() = assertEquals(3493, day.part2())
}
